constraint/pkg/client/drivers/local/driver.go

```diff
@@ -71,14 +73,13 @@ type Driver struct {
 
 	// clientCertWatcher is a watcher for the TLS certificate used to communicate with providers.
 	clientCertWatcher *certwatcher.CertWatcher
-}
 
-// RegoEvaluationMeta has rego specific metadata from evaluation.
-type RegoEvaluationMeta struct {
-	// TemplateRunTime is the number of milliseconds it took to evaluate all constraints for a template.
-	TemplateRunTime float64 `json:"templateRunTime"`
-	// ConstraintCount indicates how many constraints were evaluated for an underlying rego engine eval call.
-	ConstraintCount uint `json:"constraintCount"`
+	// gatherMetrics controls whether the driver attempts to gather any metrics around its API calls
+	gatherMetrics bool
+
+	// StatsEntries returns the current statistics observed since the last API call. At present
+	// StatsEntries are only gathered on the Query() API.
+	StatsEntries []*instrumentation.StatsEntry
```

Comment:
We could make this unexported but then we'd need some brittle reflection to access it. One pro of going that way would be that CF users wouldn't be tempted to use this.